libraries/ui-library/src/utils/execution-control.ts

```diff
@@ -33,8 +31,10 @@ export const debounce = (callback: Function, timeout = DEFAULT_DEBOUNCE_FAST) =>
  * @param timeout
  */
 export const debounceEvent = (event: EventEmitter, timeout: number): EventEmitter => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Hm, I think it would be nicer to introduce a custom type instead of "as any" and the comment:

```typescript
interface DebounceEventEmitter extends EventEmitter {
  _original?: EventEmitter;
  emit: (_: unknown) => CustomEvent;
}

export const debounceEvent = (event: EventEmitter, timeout: number): DebounceEventEmitter => {
  const original = ((event as DebounceEventEmitter)._original || event) as DebounceEventEmitter;
  ...
```